Userland/Libraries/LibPDF/Filter.cpp

```diff
@@ -305,20 +312,25 @@ PDFErrorOr<ByteBuffer> Filter::decode_ccitt(ReadonlyBytes, RefPtr<DictObject> de
             damaged_rows_before_error = decode_parms->get_value(CommonNames::DamagedRowsBeforeError).get<int>();
     }
 
-    // FIXME: Do something with these.
-    (void)require_end_of_line;
-    (void)encoded_byte_align;
-    (void)columns;
-    (void)rows;
+    // FIXME: This parameter seems crucial when reading its description, but we still
+    //        achieve to decode images with it. Figure out what to do with it.
```

Comment:
Nit: this comment is confusing me. Do you mean that we still manage to decode images without it?

Reply:
I understood it to mean that we manage to decode (some?) images that have it set even though we ignore it.